test/fuzzing/ordering_service_fixture.hpp

```diff
@@ -13,12 +13,11 @@
 
 #include "backend/protobuf/proto_transport_factory.hpp"
 #include "backend/protobuf/transaction.hpp"
+#include "cryptography/public_key.hpp"
```

Comment:
It does not look like it is used in this file. If not, it can be removed.

Reply:
After discussion with @lebdron we decided to leave it as is for now.